extensions/scheduler/runtime/src/main/java/io/quarkus/scheduler/runtime/util/SchedulerUtils.java

```diff
@@ -47,28 +66,64 @@ public static long parseEveryAsMillis(Scheduled scheduled) {
      */
     public static String lookUpPropertyValue(String propertyValue) {
         String value = propertyValue.trim();
-        if (!value.isEmpty() && isConfigValue(value)) {
-            value = ConfigProviderResolver.instance().getConfig().getValue(getConfigProperty(value), String.class);
+        if (!value.isEmpty()) {
```

Comment:
I think that we should avoid the lookup completely if no config syntax is used...

Reply:
Done. I don't compile the expression anymore to figure out if it uses variables. Else the optimization would be pointless.